src/components/AllCitiesCategoryContainer/AllCitiesCategory/AllCitiesCategory.js

```diff
@@ -1,40 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import flatten from 'lodash/flatten';
 import PageWrapper from '../../PageWrapper/PageWrapper';
 import PageBanner from '../../PageBanner/PageBanner';
 import SubCategorySummary from '../../SubCategorySummary/SubCategorySummary';
-import getSubCategorySectionId from '../../../helpers/getSubCategorySectionId';
+import SubCategoryDetails from '../../SubCategoryDetails/SubCategoryDetails';
+import CityColumnChart from '../../CityColumnChart/CityColumnChart';
 import { INDICATORS } from '../../../constants';
+import style from './AllCitiesCategory.scss';
 
-const AllCitiesCategory = props => (
-  <PageWrapper categoryId={props.category.id}>
-    <PageBanner
-      colorName={props.category.colorName}
-      description={props.category.description}
-      indicator={props.category.heroIndicatorId}
-      title={props.category.name}
-      cities={props.cities}
-    />
+const AllCitiesCategory = (props) => {
+  const isHeroInSubCategory = props.category.subCategories.find(sc => (
+    flatten(sc.charts.map(chart => chart.indicatorIds)).includes(props.category.heroIndicatorId)
+  )) !== undefined;
 
-    {props.category.subCategories.map(subCategory => (
-      <SubCategorySummary
-        key={subCategory.name}
-        {...subCategory}
+  return (
+    <PageWrapper categoryId={props.category.id}>
+      <PageBanner
         colorName={props.category.colorName}
+        description={props.category.description}
+        indicator={props.category.heroIndicatorId}
+        title={props.category.name}
         cities={props.cities}
       />
-    ))}
 
-    {props.category.subCategories.map(subCategory => (
-      <div
-        key={subCategory.name}
-        id={getSubCategorySectionId(subCategory.name)}
-      >
-        {subCategory.name} chart section placeholder
-      </div>
-    ))}
-  </PageWrapper>
-);
+      {props.category.subCategories.map(subCategory => (
+        <SubCategorySummary
+          key={subCategory.name.replace(' ', '-')}
+          {...subCategory}
+          colorName={props.category.colorName}
+          cities={props.cities}
+        />
+      ))}
+
+      {isHeroInSubCategory || <div className={style.heroWrapper}>
```

Comment:
This hero chart might be redundant, not 100% sure if hero charts will always be in a subcategory, yet. Once data task is done we might want to get rid of this bit.

Reply:
Oh yep, that is nicer.